big-practice/src/js/dashboard.js

```diff
@@ -37,4 +39,8 @@ const renderTable = () => {
     .catch((error) => alert('An error occurred while getting movie', error));
 };
 
+addBtn.addEventListener('click', () => {
+  modal.classList.add('form-appears');
```

Comment:
Should be more specific for modal instead of form. Because you open a modal, model content can be anything, not only for form.
Try `model-show`, `show`, `model-open` or something like that